queries/usd/indents.scm

```diff
@@ -0,0 +1,19 @@
+[
+  (block)
+  (dictionary)
+  (list)
+  (list_proxy)
+  (metadata)
+  (timeSamples)
+  (tuple)
+  (variant_set_definition)
+] @indent.begin
+
+(block "}" @indent.end)
```

Comment:
Seems? to be missing `@indent.branch` for all of these`

Reply:
Yep, that was a good catch. Just verified it on a test file. USD comes with a command that effectively formats the USD code. I created [this fairly complex USD file](https://gist.github.com/ColinKennedy/1997d1f0850031d22245f529cd92872f), intentionally messed up the indents, and my current indent logic doesn't match what USD's command outputs. But adding `@indent.branch` results in a perfect diff. Will fix!

Reply:
GitHub doesn't mark this out of date because I changed lines below but we should be good to go now. At my current place it's common practice for the reviewee to mark conversations as resolved but since I'm not sure that's common in OSS repos I will let a maintainer mark it instead